### PR TITLE
chore: replace eslint-config-ovh by eslint-config-airbnb-base

### PR DESCRIPTION
**Pull request by antleblanc**
### Description of the Change

db342a2 — chore: replace eslint-config-ovh by eslint-config-airbnb-base

### Benefits

Aligned  with [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript).

ref: MFRWW-366

### CODE REVIEW

**Comment by FredericEspiau** [comment]
*sigh* 827 files, are those mostly `eslint --fix` modifications ?

**Comment by antleblanc** [comment]
Yes! I just had to fix bunch of errors :boom: such as `max-len` or `no-param-reassign`.

File: Gruntfile.js [hunk 1]
```diff
@@ -180,9 +180,6 @@ module.exports = function (grunt) {
         //##            For ES6 support                                                        ##
         //#######################################################################################
         babel: {
-            options: {
```
Comment (FredericEspiau) [inline Gruntfile.js]: :+1:

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (FredericEspiau) [inline .babelrc]: :+1:

File: .babelrc [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+    "presets": ["es2015"]
```
Comment (FredericEspiau) [inline .babelrc]: Maybe in another PR we could use https://babeljs.io/docs/plugins/preset-env/ ?

**Pull request merged**